src/point-cloud-octree.ts

```diff
@@ -451,7 +452,14 @@ export class PointCloudOctree extends PointCloudTree {
     renderer.state.setBlending(NoBlending);
 
     renderer.setRenderTarget(pickState.renderTarget);
+
+    // Save the current clear color and clear the renderer with black color and alpha 0.
+    const oldClearColor = renderer.getClearColor();
+    const oldClearAlpha = renderer.getClearAlpha();
+    renderer.setClearColor(new Color(0, 0, 0), 0);
```

Comment:
Maybe we can have a `BLACK` constant somewhere and just pass it in instead of building a new object each time.

Reply:
Added to the constants file, let and started using it. Let me know if this is what you had in mind. And resolve if so.